docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,161 @@
+4. Click **Execute**
+![find example step 3 and step 4](../_static/images/howto_guides/api_gui/find_example_step3and4.png)
+5. View the results in a json format, available to download by clicking **Download**, or copy the results by clicking the clipboard icon in the bottom right corner of the response. In this case two studies were retrieved.
+![find example step 5](../_static/images/howto_guides/api_gui/find_example_step5.png)<br/>
+<br/>
+
+- Note that a curl request and request url are provided as well for command line usage:
+![find example note](../_static/images/howto_guides/api_gui/find_example_note.png)
+
+#### ___Metadata___ Endpoints
+
+The [metadata endpoints](https://api.microbiomedata.org/docs#/metadata) can be used to get and filter metadata from collection set types (including studies, biosamples, activites, and data objects as discussed in the __find__ section), as well as validate and submit updates to existing metadata to the data portal as a spreadsheet or json file. 
+
+The syntax for the filter parameter of the __metadata__ endpoints is slightly different than that of the __find__ endpoints using [MongoDB-like anguage querying](https://www.mongodb.com/docs/manual/tutorial/query-documents/) instead of the compact syntax the __find__ endpoints use. The applicable parameters of the __metadata__ endpoints, with acceptable syntax and examples are in the table below.
+
+| Parameter | Description | Syntax | Example |
+| :---: | :-----------: | :-------: | :---: | 
+| collection_name | The name of the collection to be queried. For a list of collection names please see the [Database class](https://microbiomedata.github.io/nmdc-schema/Database/) of the NMDC Schema | String | `biosample_set` |
+| filter | Allows conditions to be set as part of the query, returning only results that satisfy the conditions | [MongoDB-like query language](https://www.mongodb.com/docs/manual/tutorial/query-documents/). All strings should be in double quotation marks. | `{"lat_lon.latitude": {"$gt": 45.0}, "ecosystem_category": "Plants"}` | 
+| max_page_size | Specifies the maximum number of documents returned at a time | Integer | `25`
+| page_token | Specifies the token of the page to return. If unspecified, the first page is returned. To retrieve a subsequent page, the value received as the `next_page_token` from the bottom of the previous results can be provided as a `page_token`. ![next_page_token](../_static/images/howto_guides/api_gui/metadata_page_token_param.png) | String | `nmdc:sys0ae1sh583`
+| projection | Indicates the desired fields to be included in the response. Helpful for trimming down the returned results | Comma separated string of field names that correspond to a `collection_name`. | `name, ecosystem_type` |
```

Comment:
The wording of "field names that correspond to a `collection_name`" was a little confusing to me. Here's an alternative I find a little more clear, but not as concise.

```diff
- Comma separated string of field names that correspond to a `collection_name`
+ Comma-separated list of fields that belong to the documents in the collection being queried
```

Reply:
Thank you! I added your suggestion. I also change "fields" to "attributes". I was trying to stay consistent with what we call these and I think I landed on attributes, but there might be some "fields" sprinkled in there. See commit [f0f351a](https://github.com/microbiomedata/NMDC_documentation/pull/27/commits/f0f351ab5090198cb21d375dfc9fb6e503f6376c)